### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by YevhenOstrovskiy**
[DEMO LINK](https://YevhenOstrovskiy.github.io/react_todo-app-add-and-delete/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,24 +1,69 @@
...
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <InputForm onSubmit={addTodo} onCompleted={changeTodo} />
+
+        <TodoList status={status} />
+
+        {/* Hide the footer if there are no todos */}
+        {todos.length && (
+          <Filter
```
Comment (Xilston) [inline src/App.tsx:49]: Why did you name Footer as Filter? change naming.

File: src/components/TodoItem/TodoItem.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    event.preventDefault();
+
+    try {
+      const deleteId: number = todo.id;
+
+      removeTodo(deleteId);
+    } catch (error) {
+      throw new Error();
+    } finally {
+      console.log(`delete Todo by id  ${todo.id}`);
```
Comment (Xilston) [inline src/components/TodoItem/TodoItem.tsx:25]: remove all console log in final code.

File: src/types/Context.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { Todo } from './Todo';
+
+export interface Context {
```
Comment (Xilston) [inline src/types/Context.ts:3]: Change naming, `Context` - not informative name.

**Review by Xilston — APPROVED** [review]
Gooj job! I see that you are not finished yet, I'll aproove it for mentor's review.

**Review by olya-shyian — APPROVED** [review]
I approve without a code review, I will do it already in the 3rd task.

p.s please fix your linter failed test and read comments from your colleague

**Pull request closed without merging**